From eda4d7d259ddb1c1db2074712f62c5c5c73135f8 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Tue, 29 Dec 2020 13:21:52 +0200
Subject: [PATCH] [SW-27979] preboot: increase sram section size

.sram section size of 48KB is no longer enough. Increasing to 52KB.

Change-Id: I81ea05d264fc42a21642f5c3499a5ea0a699997d
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 61d9934..743d8a3 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -8,7 +8,7 @@
 #define COMMON_GAUDI_H_
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0xC000		/* 48KB */
+#define PREBOOT_SIZE			0xD000		/* 52KB */
 #define SRAM_SIZE			0x1400000	/* 20MB */
 
 #define HL_SP_SRAM_BASE			0x7FFBFE0000
